src/wagtail_live/publishers/django_channels/publisher.py

```diff
@@ -0,0 +1,21 @@
+from asgiref.sync import async_to_sync
+from channels.layers import get_channel_layer
+
+from wagtail_live.publishers.websocket import BaseWebsocketPublisher
+
+
+class DjangoChannelsPublisher(BaseWebsocketPublisher):
+    """Django channels publisher."""
+
+    def publish(self, channel_id, renders, removals):
+        """Sends updates to the room group corresponding to channel_id."""
+
+        channel_layer = get_channel_layer()
+        group_name = f"liveblog_{channel_id}"
+        message = {
+            "type": "new_update",
```

Comment:
what is the difference between an update and a new update? ;-)

Reply:
I don't understand your question.

Reply:
Just a naming thing, not referring to somewhere else :-) I meant, why do you call it "new_update" instead of just "update". Because an update is supposed to be new anyway, right? So "new" seems redundant :-)

Reply:
Ah alright! And I also name the method update on the consumer. Good!

Reply:
Done!